### DISCUSSION TITLE
Orthographic camera

### DISCUSSION THREAD

**Discussion by Unwarped** [jnsmalm/pixi3d#49] [Q&A] [answered]
How can I use an orthographic camera? Thanks!

**Comment by Unwarped** [comment, marked as answer]
I found it!
control.camera._orthographic = true;

**Reply by jnsmalm** [reply]
You can also do PIXI3D.Camera.main.orthographic = true and control the size with PIXI3D.Camera.main.orthographicSize

**Reply by jnsmalm** [reply]
Btw you can find the API reference at https://api.pixi3d.org

**Reply by Unwarped** [reply]
Yes I see what you mean about the main camera. This is very useful thank you jnsmalm! Very handy detailed reference pages.
I will do some tests switching between multiple camera views.
OrthographicSize I will try changing that on mouse scroll events for a zoom in feature with locked isometric rotation!

**Reply by Unwarped** [reply]
Off topic but I have one more question.
Is there a method to determine mouse click position? For example if I have a ground plane with .interactive = true. 
I found the functions screenToRay, screenToWorld, worldToScreen and tried passing in the screen MouseX, MouseY but getting weird results.
I'm trying to place a mesh where you click on the ground or later have a mesh walk to it.
Again thank you for all of your help!
